fuel/src/main/kotlin/com/github/kittinunf/fuel/core/requests/UploadTaskRequest.kt

```diff
@@ -14,8 +14,8 @@ class UploadTaskRequest(request: Request) : TaskRequest(request) {
 
     val BUFFER_SIZE = 1024
 
-    val CRLF = "\\r\\n"
-    val boundary = System.currentTimeMillis().toHexString()
+    val CRLF = "\r\n"
+    val boundary = request.httpHeaders["Content-Type"]!!.split("=", limit=2)[1]
```

Comment:
I am a bit hesitant about using `!!`. It would crash if `httpHeaders` doesn't contain `"Content-Type"`. Can you do sth about that?